exporter/awsemfexporter/datapoint.go

```diff
@@ -105,6 +105,28 @@ type summaryMetricEntry struct {
 	count uint64
 }
 
+type dataPointSplit struct {
+	values     []float64
+	counts     []float64
+	count      int
+	length     int
+	bucketMin  float64
+	bucketMax  float64
+	maxBuckets int
+}
```

Comment:
nit: Why do we need a new struct? Can't we reuse the `cWMetricHistogram`? It seems like most of these (aside from `length` and `maxBuckets`) map 1:1 with the fields in the `cWMetricHistogram` which is the end result. Could have the struct be something like

```suggestion
type dataPointSplit struct {
	cWMetricHistogram
	length     int
	capacity int // seems like maxBuckets is being used in this manner similar to `len` and `cap`
}
```

Reply:
Updated with the struct suggested.